app/models/game.rb

```diff
@@ -23,7 +23,6 @@ def reset_piece_players
     pieces.where(is_black: false).update_all(user_id: user_id_white)
   end
 
-  # rubocop:disable Metrics/AbcSize
```

Comment:
And this should probably come back (since in general we do want to enforce `Metrics/AbcSize` though it's impossible with this method specifically)